pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -175,13 +175,14 @@ func (alloc *allocateAction) Execute(ssn *framework.Session) {
 				}
 			}
 
-			if assigned {
-				jobs.Push(job)
-				// Handle one assigned task in each loop.
+			if !assigned {
```

Comment:
@Jeffwan , I reverted your PR. Because for training job, it need all pods to be started together; and for spark job, it requires master (driver) pod to be started firstly. If another case I missed, please let me know.

Reply:
@k82cn. Thanks! I will double check my cases.